data/tlg4019/tlg001/tlg4019.tlg001.opp-grc1.xml

```diff
@@ -64,20 +64,19 @@
         <listBibl>
           <biblStruct>
             <monogr>
-              <title xml:lang="lat">Olympiodori prolegomena et in categorias commentarium </title>
-              <editor>
-                <persName>
```

Comment:
The `<persName>` tag usage isn't incorrect — it's just hit or miss whether it appears or not. There probably should be a uniform standard.